modules/ROOT/assets/attachments/sync-gateway-metrics.yaml

```diff
@@ -49,11 +49,10 @@ paths:
       - Prometheus
       summary: Debugging/monitoring runtime stats in Prometheus format
       description: |+
-        _metrics returns a number of runtime variables that you can view for debugging or performance monitoring purposes.
+        _metrics returns a number of runtime variables that you can view for debugging or performance monitoring purposes.   The response returns the stats in Prometheus format --
 
-        The response returns the stats in Prometheus format -- see our blog on [Monitoring Sync Gateway with Prometheus and Grafana](https://blog.couchbase.com/monitoring-sync-gateway-prometheus-grafana/)
-
-        For more details on the metrics collected and reported by Sync Gateway -- see[Sync Gateway Statistics Schema](./../stats-monitoring.html).
+        - For more details on the metrics collected and reported by Sync Gateway -- see: [Sync Gateway Statistics Schema](./../stats-monitoring.html).
+        - For more on monitoring Sync Gateway with Prometheus -- see: our blog on [Monitoring Sync Gateway with Prometheus and Grafana](https://blog.couchbase.com/monitoring-sync-gateway-prometheus-grafana/)
```

Comment:
Perhaps this could be updated to the more recent one? 
https://blog.couchbase.com/monitoring-and-visualization-of-couchbase-sync-gateway-with-prometheus-and-grafana/

Reply:
Done, thanks

Reply:
Doesn't seem done here?